### ISSUE TITLE
Invite others

### ISSUE THREAD

**Issue by urwrstkn8mare** [GOG-Nebula/.github#8]

**Comment by urwrstkn8mare** [comment]
Also, if we've already forked their repo we could invite them to archive their repo and continue maintaining it on the organisation's fork if they so wish.

**Comment by UncleGoogle** [comment]
>Also, if we've already forked their repo we could invite them to archive their repo and continue maintaining it on the organisation's fork if they so wish.

Note that having original repositories on developer sites gives them more flexibility (no 3 approvals required) and also less work for us -- only the release process would require review & approval.

But we can go back to this idea after initial org goals are fulfilled.

----

I would also consider other people eg those who created important PRs for FriendsOfGalaxy plugins.

PS
I removed the milestone as inviting others does not block the external forks synchronization, it can goes separately.

PPS
damn, I can't use this github UI for managing milestones and labels, I sorry for making high notification traffic

**Comment by urwrstkn8mare** [comment]
> Note that having original repositories on developer sites gives them more flexibility (no 3 approvals required) and also less work for us -- only the release process would require review & approval. 

That's a very good point. However, my thought process was if a member of the org wanted to contribute, it would have to be a PR to the original repo which the maintainer has to approve, then create a new release, then over here we would have to review and approve. Alternatively, the maintainer could work on a feature/fix on his fork or branch of the repo (like any other contributor to a repo in this org) and then open a PR while everything else (creating releases, versioning, etc.) can be handled by the org. Additionally, other members can contribute to the repo more directly without have to rereview and approve their own work when it gets synced back. But I don't know if that thought process was right. 

> But we can go back to this idea after initial org goals are fulfilled.

Yeah we can come back to this after initial org goals. For now I think synchronisation is the best way forward.

> I would also consider other people eg those who created important PRs for FriendsOfGalaxy plugins.

Yeah hence "plugin maintainers/contributors".

> I removed the milestone as inviting others does not block the external forks synchronization, it can goes separately.

👍 

> damn, I can't use this github UI for managing milestones and labels, I sorry for making high notification traffic

Oh don't worry I don't this those things showed up as a notification for me.

**Comment by UncleGoogle** [comment]
> That's a very good point. However, my thought process was if a member of the org wanted to contribute, it would have to be a PR to the original repo which the maintainer has to approve, then create a new release, then over here we would have to review and approve. Alternatively, the maintainer could work on a feature/fix on his fork or branch of the repo (like any other contributor to a repo in this org) and then open a PR while everything else (creating releases, versioning, etc.) can be handled by the org. Additionally, other members can contribute to the repo more directly without have to rereview and approve their own work when it gets synced back. But I don't know if that thought process was right.

this is correct -- it is easier to manage one repo than manage contributions to fork thought a original repo. We will have to solve that after this problem will arise eventually :)
